Admin/MenuNodeAdmin.php

```diff
@@ -160,10 +160,10 @@ public function getFormBuilder()
      */
     public function buildBreadcrumbs($action, MenuItemInterface $menu = null)
     {
-        $menuNodeNode = parent::buildBreadcrumbs($action, $menu);
+        $menuNodeItem = parent::buildBreadcrumbs($action, $menu);
```

Comment:
its building the bradcrumb in the sonata admin in a way that reflects the menu hierarchy so that you can see where the menu item is situated. i think the confusing bit is that it builds menu nodes for the menu items that we edit...